roles/pulp_webserver/README.md

```diff
@@ -32,7 +32,7 @@ Pulp Content App. pulp_ansible has an example of such configs [here](https://git
 pulp_ansible/tree/master/pulp_ansible/app/webserver_snippets).
 
 The Nginx config provides definitions for the location of the Pulp Content App and the Pulp API as
-pulp-api and pulp-content respectively. To route the url `/pulp_ansible/galaxy/` to the Pulp API you
+pulp-api and pulp_content respectively. To route the url `/pulp_ansible/galaxy/` to the Pulp API you
```

Comment:
pulp-api and pulp_content are inconsistent here.

We still reference "pulp-content" in nginx.conf.j2 & pulp-vhost.conf.j2 as well.

It's not a big deal, but I think it's best to leave the dash.

Note: In the long run, I'd like to restructure the roles, probably by having a "pulp_common" role, that both the "pulp_api" and "pulp_content" roles depend on.